src/Uno.UI/Helpers/WinUI/CppWinRTHelpers.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Foundation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Documents;
+using Windows.UI.Xaml.Input;
+
+namespace Uno.UI.Helpers.WinUI
+{
+	internal static class CppWinRTHelpers
+	{
+		public static bool SetFocus(DependencyObject obj, FocusState focusState)
+		{
+			if (obj != null)
+			{
+				// Use TryFocusAsync if it's available.
+				if (false) //TODO:MZ: always true
+				{
+					//var result = FocusManager.TryFocusAsync(obj, focusState); //TODO:MZ:TryFocusAsync is not available in Uno!
+					//if (result.Status == AsyncStatus.Completed)
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6063676)